### ISSUE TITLE
Fix JDK10 build & Lombok

### ISSUE THREAD

**Issue by sfuhrm** [1and1/snmpman#27]
```
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 25 source files to /home/travis/build/1and1/snmpman/snmpman/target/classes
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by lombok.javac.apt.LombokProcessor to field com.sun.tools.javac.processing.JavacProcessingEnvironment.discoveredProcs
WARNING: Please consider reporting this to the maintainers of lombok.javac.apt.LombokProcessor
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by sfuhrm** [comment]
Possible fix in  1813b8d

**Comment by sfuhrm** [comment]
Fixed
